src/Fantomas.CoreGlobalTool/Program.fs

```diff
@@ -148,16 +167,25 @@ let main argv =
                 | None -> OutputPath.Notknown
 
         let inputPath =
-            let input = results.GetResult<@ Arguments.Input @>
-            let hasStdin = results.Contains<@ Arguments.Stdin @>
-            if hasStdin then
-                InputPath.StdIn input
-            elif Directory.Exists(input) then
-                InputPath.Folder input
-            elif File.Exists input && isFSharpFile input then
-                InputPath.File input
-            else
-                InputPath.Unspecified
+            let maybeInput = results.TryGetResult<@ Arguments.Input @>
```

Comment:
Interesting, by using `TryGetResult` the current workaround for version can be simplified I think.
Something for another PR but thanks for this insight.